com.unity.netcode.gameobjects/Runtime/Configuration/SessionConfig.cs

```diff
@@ -0,0 +1,58 @@
+namespace Unity.Netcode
+{
+    internal class SessionConfig
+    {
+        /// <summary>
+        /// The running list of session versions
+        /// </summary>
+        public const uint NoFeatureCompatibility = 0;
+        public const uint BypassFeatureCompatible = 1;
+        public const uint ServerDistributionCompatible = 2;
+
+        // The most current session version (!!!!set this when you increment!!!!!)
+        public static uint PackageSessionVersion => ServerDistributionCompatible;
+
+        internal uint SessionVersion;
+
+        public bool ServiceSideDistribution;
```

Comment:
Does it make sense that `ServiceSideDistribution` is in the ConnectionApproved and ConnectionRequested messages?

Reply:
It only makes sense because the ConnectionRequestMessage uses it to define the client/package version (unifying the version to a single location) and is not actually serialized where the ConnectionApprovedMessage actually does serialize it when received by the client.